### ISSUE TITLE
RTT terminal with defmt not showing (formatted) output. 

### ISSUE THREAD

**Issue by FuziNL** [probe-rs/cargo-embed#328]
The RTT terminal is not showing (proper formatted) output while using cargo embed 0.12.0. Target just runs a for-loop which prints ''Hello world" to RTT.

- Formatting set to 'Defmt': No output at all can be seen. 
- Formatting set to 'String': Output, but not properly formatted (I only see random characters, not the string sent from host). Data keeps appending at first and only timestamp (when timestamps turned on). 
- Formatting set to 'BinaryLE': I see some output, hex data does not convert to 'hello world' used as test. 

While using probe-run(0.3.3) instead of cargo embed the proper formatted output can be seen in the terminal.

**Comment by mxseev** [comment]
It looks like this bug has already been fixed, but the version on crates.io is out of date. Installing it from the repository helped me:

```
cargo install --git https://github.com/probe-rs/cargo-embed.git
```

**Comment by Yatekii** [comment]
This is because 0.12 was released before defmt 0.3 :) As @mxseev says correctly, installing from master will solve this for you :)

**Comment by FuziNL** [comment]
In that case the issue can be closed
